shared/uavobjectdefinition/actuatordesired.xml

```diff
@@ -23,5 +23,12 @@
     <field defaultvalue="0" elements="1" name="SystemIdentCycle" type="uint16" units="samples">
       <description/>
     </field>
+    <field defaultvalue="FALSE" elements="1" name="FlipOverThrustMode" type="enum" units="">
+      <description>TRUE if special "only spin some motors" behavior is requested</description>
```

Comment:
That's not very clear at all TBH.

Reply:
How good do we want descriptions to be for internal objects (e.g. not configuration)?  Is the audience end-users, developers, etc?  It's also documented some when mode is used.

Reply:
Ah, missed that it was not a setting. It's confusing because you are explaining one special mode by defining another equivalent special mode. :stuck_out_tongue: